app/assets/javascripts/geoblacklight/modules/results.js

```diff
@@ -1,76 +1,80 @@
-Blacklight.onLoad(function() {
-  var historySupported = !!(window.history && window.history.pushState);
+Blacklight.onLoad(function () {
+    var historySupported = !!(window.history && window.history.pushState);
```

Comment:
Are these just whitespace changes? If so can we move these to separate commits? 

We standardize on two spaces everywhere, it looks like this is using 4 spaces? I realize that we could probably  update the tooling to better support this. https://github.com/geoblacklight/geoblacklight/blob/master/.jshintrc#L32